tools/cli/go-whisk/whisk/api.go

```diff
@@ -152,7 +152,7 @@ func (s *ApiService) List(apiListOptions *ApiListOptions) (*RetApiArray, *http.R
     }
 
     apiArray := new(RetApiArray)
-    resp, err := s.client.Do(req, &apiArray)
+    resp, err := s.client.Do(req, &apiArray, DoNotProcessTimeOut)
```

Comment:
All of the api requests to the controller are actually action invocations, so all of the Do() calls in this module need the ExitWithErrorOnTimeout/ProcessTimeout constant.